README

```diff
@@ -13,3 +13,6 @@ svnhooks/
         Hook scripts for our Subversion repositories.  This set comes
         from the krb5 repository; krb5-appl might have a slightly
         different set of scripts.
+
+test1
```

Comment:
Commenting on lines on the pull-request diff (the "files changed" tab) are supposed to remain persistent even if the commits involved change.  "kaduk commented on an outdated diff", etc..